pkg/service/service.go

```diff
@@ -669,6 +669,30 @@ func filterServiceBackends(svc *svcInfo, onlyPorts []string) map[string][]*lb.Ba
 	return res
 }
 
+// filterPrefereredBackends returns the slice of backends which are prefered for the given service.
```

Comment:
In addition to what robot said, there's an extra `e` in `filterPrefereredBackends`

Reply:
(Ah, robot didn't comment here. There's a typo in comments `prefered` -> `preferred`.)